packages/aws-fargate/test/esm/test.js

```diff
@@ -0,0 +1,205 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { expect, assert } = require('chai');
+const { fail } = assert;
+const path = require('path');
+const constants = require('@instana/core').tracing.constants;
+
+const Control = require('../Control');
+const { expectExactlyOneMatching } = require('../../../core/test/test_util');
+const config = require('../../../serverless/test/config');
+const retry = require('../../../serverless/test/util/retry');
+
+const downstreamDummyPort = 4567;
+const downstreamDummyUrl = `http://localhost:${downstreamDummyPort}/`;
+const region = 'us-east-2';
+const account = '555123456789';
+const instrumentedContainerName = 'nodejs-fargate-test-container';
+const taskArn = `arn:aws:ecs:${region}:${account}:task/55566677-c1e5-5780-9806-aabbccddeeff`;
+const instrumentedContainerId = `${taskArn}::${instrumentedContainerName}`;
+const semver = require('semver');
+
+const containerAppPath = path.join(__dirname, './app.mjs');
+const instanaAgentKey = 'aws-fargate-dummy-key';
+const nodeMajorVersion = process.versions.node;
+const minNodeJsVersion = '14.0.0';
+const maxNodeJsVersion = '18.17.1';
```

Comment:
TBD: Why is this hardcoded in this test? Looks very hard to maintain

Reply:
We discussed about this, and will move this version check to a common module dedicated esm version checks ✌️